### DISCUSSION TITLE
supabase auth emails landing in spam folder

### DISCUSSION THREAD

**Discussion by laurentS** [supabase/supabase#674] [Questions] [answered]
Hi all,
Most of the auth emails (signup/magic link...) I am sending to myself while developing on supabase land in my spam folder. If it's just me, I'll ask my email provider about it, but if other people are having the same issue, then maybe something can be done on the sending end?

**Comment by kiwicopple** [comment]
We did see another dev having issues here. Did you also have this problem when using your own SMTP server?

We're using AWS SES, so we'll check with them why this could be the case (everything seems to be configured fine on our side)

**Reply by laurentS** [reply]
I've not tried using my own SMTP, but will do and report back. I guess if I'm emailing myself with it, it should be fine...
I'll also contact my provider, just to mention that these emails should not be spammed.

**Reply by laurentS** [reply]
For the record, I contacted my email provider about the issue, and they tweaked their spam filters, now it's all fine. So I guess it's all good on your end :)

**Reply by n1ghtmare** [reply]
I would just like to point out that I'm having the same issue, when I check *gmail* the login emails from supabase land in the spam folder.

**Comment by thorwebdev** [comment, marked as answer]
One recommendation is to update and personalise the email templates for your project at https://app.supabase.io/project/xxx/auth/templates. Using a third-party email server such as SendGrid can also help: https://github.com/supabase/supabase/discussions/1666#discussioncomment-837744
